src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -682,6 +682,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Intrinsics\Vector256_1.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Intrinsics\Vector256DebugView_1.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Intrinsics\X86\Enums.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\Loader\AssemblyLoadContext.cs" />
```

Comment:
Could you please make this `Condition="'$(TargetsCoreRT)' != 'true'"` for now.

Reply:
done